src/esm_runscripts/oasis.py

```diff
@@ -154,30 +156,27 @@ def add_coupling(
                 trafo_line += " BLASOLD"
                 coefficient = transformation["preprocessing"][pre].get("xmult", None)
                 if not coefficient:
-                    print(
+                    logger.error(
                         "xmult needs to be defined for preprocessing BLASOLD",
-                        flush=True,
                     )
                     sys.exit(2)
                 add_scalar = transformation["preprocessing"][pre].get(
                     "add_scalar", None
                 )
                 if not add_scalar:
-                    print(
+                    logger.error(
                         "add_scalar needs to be defined (0 or 1) for preprocessing BLASOLD",
                         flush=True,
                     )
-                    sys.exit(2)
```

Comment:
Does `logger.error` raises such an `sys.exit(2)` or is it obsolete here? 
I am asking because there are other `logger.error` calls still followed by such a `sys.exit(2)` call.

Reply:
Fixed!